### ISSUE TITLE
A data specification for psychological datasets

### ISSUE THREAD

**Issue by mekline** [MozillaFestival/open-leaders-6#15]
Project Lead: @mekline
Mentor: @aath0

Welcome to OL6, Cohort B! This issue will be used to track your project and progress during the program. Please use this checklist over the next few weeks as you start Open Leadership Training :tada:.

***
Before Week 1 (Sept 11): Your first mentorship call
- [ ] Complete the OLF self-assessment ([online](https://mzl.la/olf-online-assessment), [printable](https://mzl.la/olf-self-assessment)). If you're a group, each teammate should complete this assessment individually. This is here to help you set your own personal goals during the program. No need to share your results, but be ready to share your thoughts with your mentor.
- [ ] Make sure you know when and how you'll be meeting with your mentor.

Before Week 2 (Sept 18): First Cohort Call (Open by Design)
- [ ] Complete Assignment ['Write Your Vision or Mission Statement'](https://mozilla.github.io/open-leadership-training-series/articles/introduction-to-open-leadership/stating-your-project-vision/). Leave a comment on this issue with your draft vision statement & be ready to share this on the call.
- [ ] Watch ['Mozilla & Working Open'](https://youtu.be/quKdaqlR_9w).
- [ ] Check the [Syllabus](https://mzl.la/ol6-syllabus) for notes and connection info for all the cohort calls.

Before Week 3 (Sept 25): Mentorship call
- [ ] Look up two other projects and comment on their issues with feedback on their vision statement.
- [ ] Complete your Open Canvas ([instructions](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/develop-an-open-project-strategy-with-open-canvas/), [canvas](https://goo.gl/to6PYn)). Comment on this issue with a link to your canvas.
- [ ] [Start your Roadmap](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/start-your-project-roadmap/). Comment on this issue with your draft Roadmap.

Before Week 4 (Oct 2): Cohort Call (Build for Understanding)
- [ ] Look up two other projects and comment on their issues with feedback on their open canvas.
- [ ] [Pick an open license](https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/sharing-your-work-in-the-open/) for the work you're doing during the program.
- [ ] Use your canvas to [start writing a README](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/), or landing page, for your project. Link to your README in a comment on this issue.

This issue is here to help you keep track of work during the first month of the program. Please refer to the [OL6 Syllabus](https://mzl.la/ol6-syllabus) for more detailed weekly notes and assignments past week 4.

**Comment by mekline** [comment]
Draft vision statement: I’m working with researchers in psychology (and experts in metadata/data standards) to define a common, machine-readable structure for sharing psychological datasets so that researchers can make their open (or protected) data discoverable, and develop new tools and standards in a common format. 

I’m working open because while psychologists (and ‘nearby’ social sciences) have developed common standards for how we share *papers* (i.e. APA format), we lack a consensus model of what it means to share a *dataset* well. For this data standard to be successful, it needs to meet the needs of working researchers, so their early involvement is critical to getting it right.

**Comment by brownsarahm** [comment]
Your project is really exciting, your goal is concrete and your audience is clear. 

Some questions:
 - what type(s) of psychology data? neuroimaging, for example, has well developed standards
 - will your standard address the data itself only or the data and metadata?
 - what types of tool & standard development do you want the standard to support? (eg, compatibility with common analysis kits, supporting good data provenance, etc)
- machine readable in what sense? eg compatible with specific software or general purpose scientific computing resources?

Some additional resources that come to mind related to your project and that may clarify what I mean based on the questions:
 - [Datasheets for Datasets](https://arxiv.org/pdf/1803.09010.pdf) a more general aim for a standard qualitative description to accompany datasets to promote responsible use of them with a heavy ML lean as the target users of the data
 - [Data Organization in Spreadsheets for Social Sciences](https://datacarpentry.org/spreadsheets-socialsci/) a _beta_ lesson on how to organize data for later analysis using a programming language (R or Python) it's also fully open source and may be useful as a resource and/or you may learn things that help improve that.  It's repo could also be a source of more community for your project.

**Comment by mekline** [comment]
@brownsarahm I was just about to email you, believe it or not! Your comment about tooling for *small* projects especially caught me - there's both lots of potential and lots of difficulty in the 'long tail', and I would really like to see psychology moving from 'just share your data' (a critical initial policy move) to actually building resources that are interoperable & discoverable. 

I'm working with a couple of the people from BIDS (neuroimaging standard), who have been really helpful in thinking about how to carve off a manageable and useful chunk of this problem: in this case, this means focusing on psych data (or even other social science) in the very general sense - we think we're going to include a standard name for the 'person/subjectID' since this is such a central organizing variable, but beyond that won't work on nomenclature or the heavier org. structure that fMRi data requires. My hope is that if we can get a skeleton, we'll be able to support more domain-specific standards for e.g. reaction time, standard measures, etc. (In line with the BIDS extensions.) 

The general goal is a platform-agnostic data standard (with a 'wish list' of packages for various languages, example datasets etc. that we can work on building out) that funnels the carpentry-type principles (the WHY of good data management) into a common format, so that people can not only work with their own code, but also make friends with other datasets and tools. 

I see you're in Providence, I'm nearby - any interest in meeting at some point before MozFest?

**Comment by mekline** [comment]
And thank you for the resources! I'd seen the second but not the first, adding to my list!

**Comment by brownsarahm** [comment]
Yes! I actually won't make it to MozFest, but I'd like to meetup.

**Comment by majohansson** [comment]
Nice project and nice vision statement. We have some similar needs in the field of epidemiology. One concept I have found helpful is Minimum Information Requirements/Standards. That doesn't help much with the format part, but does provide some help for deciding what the format should be able to accommodate. Just one suggestion for the vision statement: maybe you can identify the end game benefit, e.g. better reproducibility, less duplication, and better science?

**Comment by mekline** [comment]
[Open Canvas Link](https://docs.google.com/presentation/d/1GQUpUPL3dHGc-Eb_3dL6WcXnA4hXpUanjAc8jUp16S0/edit?usp=sharing)

**Comment by fraguada** [comment]
@mekline looking through the open canvas and the project seems to have a lot of similar goals to [opening the healthcare data landscape #94](#94) by @ju-liem. I don't know much about this industry and any specific standards they might have for data. Are there working groups or conferences where contributors might attend? Could be a good channel for finding contributors.
